test/functional/p2p_invalid_messages.py

```diff
@@ -206,10 +206,10 @@ def test_msgtype(self):
     def test_large_inv(self):
         conn = self.nodes[0].add_p2p_connection(P2PInterface())
         with self.nodes[0].assert_debug_log(['Misbehaving', 'peer=4 (0 -> 20): message inv size() = 50001']):
-            msg = messages.msg_inv([messages.CInv(1, 1)] * 50001)
+            msg = messages.msg_inv([messages.CInv(messages.MSG_TX, 1)] * 50001)
```

Comment:
Caught my eye because it has a `messages.` in front... Would it make sense to explicitly list imports at the top of `p2p_invalid_messages.py` like all the other files do? Just quickly looking for `messages` in the file, it doesn't seem like too long of a list:

`from test_framework.messages import ser_string, msg_ping, msg_inv, msg_getdata, msg_headers, CInv, CBlockHeader, MSG_TX`

Reply:
@gzhao408: Thanks for reviewing! The idea was to keep the changes as small as possible to make reviewing easier, i.e. that on every line touched, only the proclaimed magic number elemenations happen and nothing more. Looking at `p2p_invalid_messages.py` though, I agree that it could be fit in this PR, as there are not too many changes needed. Will add a commit doing that.